package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "raspi-live",
+  "name": "raspi-live-v2",
```

Comment:
Version revisions aren't done in the package.json `name` field. Please revise the `version` field instead with a patch update.

Reply:
No problem!